definitions/variable/technology/electricity-expansion.yaml

```
@@ -25,3 +25,125 @@
     description: Removed capacity for the electricity network,
       this capacity is substracted from the existing capacity
     unit: MW
+
+- Network|Electricity|Capacity|Expansion|End Year:
+    description: The additional Capacity required by the expansion of the existing grid
+        to fully cover the demand at the end of the study.
+    unit: kW
+
+- Solar PV|Standalone|Capacity|Expansion|End Year:
+    description: The additional Capacity required by the expansion of the standalone Solar PV 
+        to fully cover the demand at the end of the study.
+    unit: kW
+
+- Solar PV|Mini Grid|Capacity|Expansion|End Year:
+    description: The additional Capacity required by the expansion of the PV minigrid
+        to fully cover the demand at the end of the study.
+    unit: kW
+
+- Hydro|Mini Grid|Capacity|Expansion|End Year:
+    description: The additional Capacity required by the expansion of the hydropower mini grid
+        to fully cover the demand at the end of the study.
+    unit: kW
+
+- Wind|Mini Grid|Capacity|Expansion|End Year:
+    description: The additional Capacity required by the expansion of the existing wind power mini
+        to fully cover the demand at the end of the study.
+    unit: kW
+
+- Network|Electricity|Expansion Cost:
+    description: The investment required by expanding the electric grid to reach the electrification target
+      at the end of the study.
+    unit: us$
+
+- Network|Electricity|New Grid Cost:
+    description: The investment required by new grid to reach the electrification target
+      at the end of the study.
+    unit: us$
+
+- Solar PV|Standalone|Expansion Cost:
+    description: The investment required by expanding standalone solar PV to reach the electrification target
+      at the end of the study.
+    unit: us$
+
+- Solar PV|Mini Grid|Expansion Cost:
+    description: The investment required by expanding mini grid solar PV to reach the electrification target
+      at the end of the study.
+    unit: us$
+
+- Hydro|Mini Grid|Expansion Cost:
+    description: The investment required by expanding the hydro power mini grid to reach the electrification target
+      at the end of the study.
+    unit: us$
+
+- Wind|Mini Grid|Expansion Cost:
+    description: The investment required by expanding the wind power mini grid to reach the electrification target
+      at the end of the study.
+    unit: us$
+
+- Expansion Cost|Total:
+    description: The total investment required by expanding to reach the electrification target
+      at the end of the study.
+    unit: us$
+
+- Population|Connected|Total:
+    description: Total predicted populations with electricity
+    unit: people
+
+- Population|Connected|Grid|Existing:
+    description: Total predicted populations with electricity from existing grid
+    unit: people
```

If this is in the millions I would suggest using
```suggestion
    unit: million
```
otherwise since it's a number I would suggest going fully dimensionless, i.e. using `unit: `.